R/lint.R

```diff
@@ -242,10 +242,16 @@ lint_package <- function(path = ".", ...,
     dots <- dots[-1L]
   }
 
+  if (length(path) > 1L) {
+    stop("Only linting one package at a time is supported.")
+  }
   pkg_path <- find_package(path)
 
   if (is.null(pkg_path)) {
-    warning("Didn't find any R package searching upwards from '", path, "'.")
+    warning(sprintf(
+      "Didn't find any R package searching upwards from '%s'.",
+      if (path == ".") normalizePath(path) else path
```

Comment:
Do we have tests for both statements here?

Reply:
Is there any benefit in only conditionally normalizing?

Reply:
I was thinking lint_package("lintr") would be easier to read than /long/path/to/where/I/have/lintr. but no strong preference.

Reply:
OTOH in the error condition, the full path avoids any working directory related confusion.

Reply:
OK, removed the conditional